articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -0,0 +1,222 @@
+   ![!Notification showing VM deployment in progress](images/azsp_createvm_progress.png)
+
+## Create the local network gateway
+
+The local network gateway refers to your on-premises network. The following steps outline how to create a local network gateway:
+
+1. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azsp_newmenu.png)
+
+2. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-browser-new-networking.png)
+
+3. In the **Featured** section, select **Local network gateway**.
+
+   ![Create new networking resource](images/azs-browser-networking-create.png)
+
+4. In the **Create local network gateway** blade, enter the following information:
+
+   **Name** - The name of the local network gateway
+
+   **IP address** - This is the public IP address of the VPN device that you want Azure to connect to. Specify a valid public IP address.
+
+   **Address space** - One or more IP address ranges (in CIDR notation) that define your local network's address space. For example: 192.168.0.0/16. If you plan to use this local network gateway in a BGP-enabled connection, then the minimum prefix you need to declare is the host address of your BGP Peer IP address on your VPN device.
+
+   **Configure BGP settings** - Use only when configuring BGP.
+
+   **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   **Location** - This will be "frn00006", which is the location of the Azure Stack.
+
+   ![Create new local network gateway](images/azs-browser-create-Lnetwork-gateway.png)
+
+5. Click **Create**.
+
+## Create the VPN connection
+
+Create the Site-to-Site VPN connection between your virtual network gateway and your on-premises VPN device:
+
+1. Navigate to your virtual network gateway by clicking on **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
+
+2. Select your virtual network gateway from the list.
+
+3. In the blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+4. Click the **+Add** button.
+
+5. In the **Add connection** blade, enter the following information:
+
+   **Name** - The name of the connection.
+
+   **Connection type** - Select "Site-to-site (IPSec)".
+
+   **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   **Local network gateway**- Select the local network gateway you created earlier.
+
+   **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. The same shared key must be used in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.

```

Comment:
Active: You must use the same shared key in both the virtual network and local network gateways.

Reply:
Fixed